### PR TITLE
Docs update for Yamuna protocol upgrade

### LINKED ISSUE #173: Concept Validator: update legacy command-line flag name `genEnodeProof` to `genOwnershipProof`
**Describe the bug**

The Concept Validator page has in section [P2P node keys: autonityKeys](https://docs.autonity.org/concepts/validator/#p2p-node-keys-autonitykeys)  an instance of the legacy name for the `genOwnershipProof` command-line option of `genEnodeProof`. This needs correcting to `genOwnershipProof`.

>To generate the proof of enode ownership required for validator registration. The proof is generated using the [genEnodeProof](https://docs.autonity.org/reference/cli/#command-line-options) command-line option of the Autonity Go Client.

Also, the notebox below refers to using the `bootnode` utility to generate the enode url

```
::: {.callout-note title="Note" collapse="false"}
The private key can be used by Autonity’s `bootnode` utility to derive the hex string used in the `enode` URL. (See Networking Options  `nodekey` and `nodekeyhex` in [Autonity command-line options](/reference/cli/#usage) and, for reference,  the ethereum stack exchange article [how to produce enode from node key](https://ethereum.stackexchange.com/questions/28970/how-to-produce-enode-from-node-key).)
:::
```

This should be updated to clarify that the first 64 characters of the private key in the `autonitykeys` file needs to be passed in as argument else it will fail (`autonitykeys` is 128 as it contains node key concatenated withconsensus key)

**Tasks**

- [x] correct `genEnodeProof` to `genOwnershipProof` here and check throughout docs
- [x] add to notebox above a clarification the first 64 characters needs to be used only.

E.g. 

>Extract the node key from your `autonitykeys` file:
>
>`head -c 64 <DIR_PATH>/autonitykeys`
>
>where `<DIR_PATH>` is the path to your `autonitykeys` file.
>
>This will print the private node key from the `autonitykeys` file to terminal.
>
>This can then be passed in as the value to the the `bootnode` utility's `-hodekey` flag or placed in a file and the file passed in as the value to the `-nodekeyhex` flag.

### LINKED ISSUE #178: Explain block proposer rewards are paid to validator account
When a validator serves as committee member, they receive their share of staking rewards to their _treasury account_. However, they receive priority fees to their _validator account_.

I could only find four references to priority fee payment in the docs, across three pages:

> The priority fee ‘tip’ that may be specified in a transaction and which is given to the block proposer as an incentive for including the transaction in a block.
https://docs.autonity.org/concepts/validator/#validator-economics

> The optional priority fee of a transaction is not included in the rewards pool but awarded to the block proposer according to the EIP 1559 transaction fee mechanism.
https://docs.autonity.org/concepts/staking/#staking-rewards

> To incentivise inclusion of a transaction in a block, EIP 1559 allows a transaction to include an optional priority fee or ‘tip’ as a reward incentive to the block proposer for including the transaction in a block.
https://docs.autonity.org/concepts/system-model/#eip-1559-transaction-fee-mechanism-tfm

> the priority fee rewards are given to the block proposer per EIP 1559. In current state, the priority fee is paid to the proposing validator entity’s account every block.
https://docs.autonity.org/concepts/system-model/#autonity-eip-1559-configuration

The first three do not explain where the fees are sent to, and therefore it is a reasonable assumption is that the fees are paid to the _treasury account_, but this is not the case.

The last instance is poorly worded, because it makes use of the ambiguous expression "proposing validator entity's account", which is unnecessarily convoluted.

It is not immediately clear to the reader if this refers to the treasury account or the validator account. Why not just say so explicitly?

In any case, from the genesis reference, we can infer that this refers to the treasury account, which is incorrect:

> | `delegationRate` | The percentage fee of staking rewards that is deducted by validators as a commission from delegated stake. The fee is sent to the validator entity's account at epoch end on reward distribution....

### LINKED ISSUE #186: Clarify Validator economics - receiving accounts for validator revenue receipts
### Description

**Context**

When a validator serves as committee member, they receive income from 4 sources:

| Income stream | Receiving account | Distribution | Description |
|:-- |:--|:--|:--|
| staking rewards | `treasury` account | epoch end | This is from their own self-bonded stake |
| commission revenue | `treasury` account | epoch end | This is commission on staking rewards for the total bonded stake bonded to the validator taken according to the validator's commission rate |
| priority fee tips | [`validator identifier`](https://docs.autonity.org/concepts/validator/#validator-identifier) account | block finalisation | When a block proposer, priority fees for transactions included in the block are transferred directly to the validator node address, the [`validator identifier`](https://docs.autonity.org/concepts/validator/#validator-identifier) account |
| slashing rewards |  `treasury` account | epoch end | As the reporting validator of an accountable fault a validator may receive slashing rewards. The staking rewards for offending validator for the epoch are forfeited and become the slashing rewards sent to the reporting validator |

This information could be more clearly stated in https://docs.autonity.org/concepts/validator/#validator-economics.

It is currently covered in 2 places:

- https://docs.autonity.org/concepts/validator/#validator-economics
- https://docs.autonity.org/concepts/accountability/#slashing-rewards

### Rationale

It should be clearly stated in https://docs.autonity.org/concepts/validator/#validator-economics. which accounts receive validator revenue.

### Implementation

Revise Validator economics.

- [x] Replace the current bullet point list with the complete table of income sources per above
- [x] Review content for clarify

Clarify block proposer rewards.

- [x] Implement changes detailed in #178

### LINKED ISSUE #188: Update Network pages for new chainID post re-genesis before PCGC R6
**Describe the edit**

There will be a testnet re-genesis before  PCGC Round 4 to bring in a protocol upgrade.

In the Networks section the testnet chain ID entries will need to be updated.

**Tasks**

| Testnet Flag |  ChainID |
| --- | --- | 
| `--bakerloo`  | 65010003 |
| `--piccadilly` | 65100003 |

**Tasks**:

- Bakerloo Testnet/ Update ChainID entry in sections:
  - [x] https://docs.autonity.org/networks/testnet-bakerloo/#bakerloo-testnet-details
  - [x] https://docs.autonity.org/networks/testnet-bakerloo/#genesis-configuration

- Piccadilly Testnet. Update ChainID entry in sections:
  - [x] https://docs.autonity.org/networks/testnet-piccadilly/#piccadilly-testnet-details
  - [x] https://docs.autonity.org/networks/testnet-piccadilly/#genesis-configuration

- Instances of ChainID elsewhere:
  - [x] Development, [Deploy smart contracts to an Autonity network with Brownie](https://docs.autonity.org/developer/deploy-brownie/) - see issue #194

### LINKED ISSUE #193: Link to page "Bug Bounty" to update
I noticed that in the page [(https://docs.autonity.org/networks/testnet-piccadilly/)] the link to the page 'Bug Bounty" redirect the user to the page "https://game.autonity.org/#tasks--points".

> image

I think it will be better to redirect the user directly to the page "Bug Bounty" (Depending on round, For Round5: https://game.autonity.org/round-5/bug-bounty).

### LINKED ISSUE #194: chainid for piccadily to update in "#setup-brownie"
In the "Setup brownie" section in the page https://docs.autonity.org/developer/deploy-brownie/#setup-brownie , the example of the commande use wrong value for the piccadilly chainID:

> image

### LINKED ISSUE #197: Slashing economics - slashed NTN handling unclear
**Describe the bug**

In slashing economics, It describes slashing reward but fails to describe what happens to slashed NTN.

This is described in section https://docs.autonity.org/concepts/accountability/#slashing-amount-calculation:

>The slashed NTN stake token is then transferred to the Autonity Protocol global treasury account for community funding.

**To Resolve**

- [x] move this or add and reword to section https://docs.autonity.org/concepts/accountability/#slashing-economics

### LINKED ISSUE #198: Oracle documentation: update oracle round price
**Describe the update**

PR https://github.com/autonity/autonity/pull/967  changes the RoundData.success semantics.  So, the docs need updating accordingly.

**Edit to make**

The merged update requires the doc update for the returned oracle round price status code: https://github.com/autonity/autonity/blob/develop/autonity/solidity/contracts/interfaces/IOracle.sol#L17, Please follow the comments:     /**
    * @notice RoundData.success Indicates the success of the round data retrieval. If `true`, the round data for the
    * requested symbol and round was successfully aggregated by the protocol. The caller should check the success code
    * before using the returned round data. If `false`, the protocol could not provide data for the requested symbol
    * and round, either because the symbol is invalid or because no data was collected for that round.
    */

### PR DESCRIPTION
**Pull request by jay-cousins**
This PR updates the docs for the Yamuna protocol upgrade.

Edits detailed beneath.

---
### Home page.

Adds hyperlinks to the Home page to the AOS and AGC repos.

---
### Edits to Concepts Validator and System model.

Edits for #173 and improves System model to better explain the new `autonitykeys` model in the section on Participants (i.e. nodes).

closes #173 

---
### Edits to Concept Accountability.

Edits for #197 in https://docs.autonity.org/concepts/accountability/#slashing-penalties table to clarify slashed NTN goes to protocol `treasury` account for community funding

closes #197 

---
### Edits to Concepts Validator, Staking, System model, and Accountability - economics sections.

Edits for #186 and #178 clarifying validator economics (incentives and disincentives), and harmonising table formatting with the parallel economics table in Accountability under slashing economics.

closes #186
closes #178 

---

### Networks

- Updated `chainId` (increments by `1`)
- Genesis configuration. Removed listings of validator enodes and replaced with link to the Piccadilly and Bakerloo Chain Config `Validators` object in AGC `/params/config.go` to simplify. 
- Fixed link to PCGC Bug bounty on Piccadilly Testnet page introduction

closes #188 
closes #193 

---

### Development, Deploy smart contracts to an Autonity network with Brownie

- Updated Piccadilly `chainId` to the new value

closes #194 

---

### EIP 1559 configuration and setting edits

- Corrected to 20M setting introduced in v0.13.0 release
- Edits to _Reference, Genesis_ and _Concepts, System model, Autonity EIP 1559 Configuration_

Commit: dba7e3f7730439d5f420910866c85421fc3c67ed.

---

### Correct _Auton Currency Unit_  to _Autonomous Currency Unit_

Edits to:

- Glossary (ACU entry)
- Reference, Genesis
- Reference, Autonity Contract Interface, Operator and Protocol Only
- Overview
- Concepts, Architecture
- Concepts, ASM, Overview

Commit: a19aa656a342ba71038670dac2bd6858c35fc385.

---

### Edits for BLS Aggregation

Edits for use of BLS to create an aggregate signature for consensus voting, replacing the previous `committedSeals` with a`quorumCertificate`:

- Concepts, System model
- Concepts, Architecture, Communication layer
- Concepts, Consensus, Tendermint BFT consensus

Commit: ced1c113b2cb0d332e4f1ec05b3f02a931cd4a42. 

---

### Node migration and validator enode update

Edits to Node operator and Validator guides to:

- Node operator, Run Autonity: move migrating a node to a separate page Migrate Autonity and edit
- Validators:  add page Migrate Validator detailing how to migrate to a new environment, update the validator's registered enode URL.

Commit: 2eae37156c989fc1658105fdb870a3869ca60149.

---

### Edits to CLI Reference

Command line (AGC):

- NETWORKING OPTIONS:  add new flag `--nogossip`

Commit: ee6a439500d8ed543dc57c8c04d34625847cf745.

(Command line (AOS) - no changes in v0.1.8)

---

### CODE REVIEW

**Comment by jay-cousins** [comment]
@aiman @yazzaoui @Jason-Zhangxin-Chen @lorenzo-dev1 @vpiyush @tbssajal 

Marking ready for review. This provides a baseline of the docs for users and node/validator operators. 

I will do a second PR with edits for:

- tokenomics, inflation mechanism
- edits from QA checks against:
  - protocol parameters
  - open issues

File: validators/migrate-vali/index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+## Migrating validator node to a new IP/Port address
+
+This guide describes how to migrate a validator node to an environment with a new IP/Port network address by updating the validator's registered enode URL.
+
+::: {.callout-warning title="Important" collapse="false"}
+Only the network connection information - IP and/or port - of a registered validator enode can be updated. You **cannot** change the validator's address (i.e. the `PUBKEY` part of the [validator enode URL](/concepts/validator/#validator-enode-url)).
+:::
+
+When performing the update the validator **must not** not be an active member of the consensus committee and should be in a [`paused`](/concepts/validator/#validator-pausing) state.
```
Comment (vpiyush) [inline validators/migrate-vali/index.md]: Only requirement is to "must noe be an active member of the consensus committee". Paused state is one of the way to achieve that but not a requirement.

File: validators/migrate-vali/index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+
+### Step 1: Pause the validator
```
Comment (vpiyush) [inline validators/migrate-vali/index.md:21]: Hence, these steps are only needed if you are the active member of the committee. and should be a recommended way so that you don't get elected as committee member while you are in the process of migration

File: node-operators/migrate-aut/index.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+::: {.callout-note title="Prerequisites" collapse="false"}
+- Ensure that the host machine meets the [minimum requirements](/node-operators/install-aut/#requirements)
+:::
+
+
+## Maintaining node identity
+
+To migrate a node to a new instance the node identity must be preserved. This is done by simply using the existing [P2P node keys: autonityKeys](/concepts/validator/#p2p-node-keys-autonitykeys) and the node's host [ip address](/node-operators/install-aut/#network) for the new node instance.  by doing this the new node instance will have the same node [identifier](/concepts/validator/#validator-identifier) address and [enode url](/glossary/#enode) as the old instance.
```
Comment (vpiyush) [inline node-operators/migrate-aut/index.md]: maintaining node identity should only be limited to using same node keys. host ip and port can now be changed by following the migration procedure. however validator must make sure that the instance ip/port is same as regsitered in enode url

File: validators/migrate-vali/index.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+Pause the validator as described in the guide [Pause as a validator](/validators/pause-vali/#pause-as-a-validator).
+
+If your validator is an active member of the consensus committed, wait for the epoch to end before proceeding further.
```
Comment (vpiyush) [inline validators/migrate-vali/index.md:25]: We can add a important note here. Validator must make sure before updating the enode. that is no longer part of the consensus committee

File: validators/migrate-vali/index.md (hunk 1 above)
Comment (jay-cousins) [reply validators/migrate-vali/index.md]: Yes. I have reworded this section so:

>When performing the update the validator **must not** not be an active member of the consensus committee. It is **recommended** that you [`pause`](/concepts/validator/#validator-pausing) your validator before migration. This is to prevent your node being elected as a committee member while you are in the process of migration.

Commit: 49f78998a49933052a54dcec360996d63e0413b5.

File: validators/migrate-vali/index.md (hunk 2 above)
Comment (jay-cousins) [reply validators/migrate-vali/index.md:21]: Reworded per preceding comment in commit 49f78998a49933052a54dcec360996d63e0413b5.

File: node-operators/migrate-aut/index.md (hunk 3 above)
Comment (jay-cousins) [reply node-operators/migrate-aut/index.md]: This page is in the _node operator_ not the _validator_ guide. `updateEnode()` has a constraint check that requires the node is a registered validator.

So you can't change the IP/Port of a plain full node.

I think the answer here is to leave that paragraph as is and to add this note box afterwards:

```
::: {.callout-note title="Additional checks if you are migrating a registered validator node" collapse="false"}
1. Make sure that your validator is not an active member of the consensus committee during the migration. It is recommended that your validator is in a [`paused`](/concepts/validator/#validator-pausing) state before beginning the migration.
2. Verify the node's new IP/Port address is the same as that in your registered enode URL. You can check this by calling `aut validator info --validator` to view the registered enode URL.

If you are operating a validator node it is possible to change your IP/Port address as described in the guide [Migrating validator node to a new IP/Port address](/validators/migrate-vali/#migrating-validator-node-to-a-new-ipport-address).
:::
```

Comment (jay-cousins) [reply node-operators/migrate-aut/index.md]: Done in commit ed9c3d7d66e1b8ec343e21f854e8c8ed20f19bbc.

File: validators/migrate-vali/index.md (hunk 4 above)
Comment (jay-cousins) [reply validators/migrate-vali/index.md:25]: Done in commit ed9c3d7d66e1b8ec343e21f854e8c8ed20f19bbc.

**Review by vpiyush — COMMENTED** [review]
I have reviewed the validator migration related changes and that looks good to me now.

**Comment by aiman** [comment]
I think we can merge for now and continue to iterate in future PR as more people review the live site.

File: concepts/consensus/pos/index.md [hunk 5]
```diff
@@ -25,7 +25,12 @@ In each consensus instance a consensus round of three voting steps is executed a
 
 If the committee member receives _2f+1_ Precommits, then by protocol the block proposal is decided and:
 
-- The block is broadcast to the network. The block proposer broadcasts the new block, its [block header](/concepts/system-model/#block-header) containing the block proposal and precommit message seals and an array of the committee members that voted for the block. Other committee members broadcast a (new) block announcement.
-- The block is committed to the ledger. Participants that receive the new block check validity by verifying the seals. Participants that receive an announcement sync to retrieve the block, verifying  by computing block state as it appends the block to the chain.
+- The block is broadcast to the network. The block proposer broadcasts the new block. Other committee members broadcast a (new) block announcement.
```
Comment (lorenzo-dev1) [inline concepts/consensus/pos/index.md:28]: by `the new block` does it mean here a block for a new height? If so I would rephrase as:

```
The decided block is broadcast to the network. The block proposer for the following height broadcasts a new proposal. 
```

If not, then there is probably an error that we need to iron out here

File: concepts/consensus/pos/index.md [hunk 6]
```diff
@@ -25,7 +25,12 @@ In each consensus instance a consensus round of three voting steps is executed a
...
+
+  ::: {.callout-tip title="Cryptographic proof of the validator quorum that agreed on the block" collapse="false"}
+This is recorded in the [block header](/concepts/system-model/#block-header) by the _proposer seal_ (seal of the committee member proposing the block) and the _quorum certificate_ (a single aggregated BLS signature of the committee members that voted and agreed on the block).
```
Comment (lorenzo-dev1) [inline concepts/consensus/pos/index.md]: The proposer seal doesn't really gives us information on the precommits quorum. I would remove that part from this box

File: concepts/system-model/index.md [hunk 7]
```diff
@@ -10,24 +10,19 @@ An Autonity network is a distributed blockchain system comprised of peer nodes r
...
+Participants are secured and uniquely identified by public key cryptography. Each participant node has cryptographic identity from keypairs, see [P2P node keys: `autonityKeys`](/concepts/validator/#p2p-node-keys-autonitykeys). The `autonitykeys` file contains two private keys that are  used to sign messages in the communication layer for transaction (`nodekey`) and consensus (`consensuskey`) gossiping. These message signatures allows cryptographic verification of message sender identity by recipient participants. The corresponding public keys are accessible to other participants and a participant can identify other participants by public key, therefore. The _public node key_ is used to derive participant identifier and network addresses. Namely, a unique network address (the [enode URL](/concepts/validator/#validator-enode-url)), and a _node address_ for the validator (the public key for the private node key).  The _public consensus key_ is used to identify the validator during consensus participation.
```
Comment (lorenzo-dev1) [inline concepts/system-model/index.md]: Not a big deal but I feel like this paragraph says multiple times the same thing

File: concepts/system-model/index.md [hunk 8]
```diff
@@ -305,19 +340,19 @@ To incentivise inclusion of a transaction in a block, EIP 1559 allows a transact
...
 
-Autonity maintains the Ethereum protocol settings for EIP 1559:
+Autonity sets an EIP 1559 configuration of:
 
-- _block gas target_ = `15M`, the pre EIP 1559 block gas limit
-- _block gas limit_ = `30M`, double the preceding block gas limit accommodating 50% elasticity
-- _base fee change denominator_ = `8`, limiting possible change in base fee to 12.5% each block.
-- _elasticity multiplier_ = `2`, setting targeted block usage at 50% of the block limit.
+- _block gas target_ = `20M`, the block gas usage targeted for a block
+- _block gas limit_ = the gas limit possible for a block is dynamic and can change (increase) by no more than 10% a block from the actual block gas usage of the parent block
```
Comment (lorenzo-dev1) [inline concepts/system-model/index.md]: Unless I am missing something, I think that the block gas limit can increase or decrease based on the parent block

File: concepts/validator/index.md [hunk 9]
```diff
@@ -45,38 +45,101 @@ Prerequisites for becoming a validator node operator are:
 The validator makes use of different accounts and private/public [key pairs](/glossary/#key-pair) for validator lifecycle management (registration, pausing, reactivation), validator identity, staking rewards, consensus participation and cryptographic security. 
 
 ### P2P node keys: autonityKeys
-The private/public key pair of the validator node.
+The private/public key pair of the validator node. The `autonitykeys` file contains the private keys for the transaction and consensus gossiping [communication layer](/concepts/architecture/#communication-layer). The keys are concatenated together to create a 128 character string:
 
-The private key is used:
+- the first 64 characters are the `node key`: used for transaction gossiping with other network peer nodes
```
Comment (lorenzo-dev1) [inline concepts/validator/index.md]: more than `transaction gossiping` I would say `transaction signing`. I saw this also in other parts of the PR

File: concepts/validator/index.md [hunk 10]
```diff
@@ -45,38 +45,101 @@ Prerequisites for becoming a validator node operator are:
...
+
+If you choose to generate the `autonitykeys` file and _do not store your key in the default location, then you must specify the path to where you are keeping your `autonitykeys` file using the `--autonitykeys` option in the run command.
+
+:::
+
+
+The private `autonitykeys` are used:
+
+- By a node for:
+  - transaction gossiping (`nodekey`), for negotiating an authenticated and encrypted connection between other network nodes at the devp2p layer in the [RLPx Transport Protocol](https://github.com/ethereum/devp2p/blob/master/rlpx.md).
```
Comment (lorenzo-dev1) [inline concepts/validator/index.md]: transaction signing

File: concepts/validator/index.md [hunk 11]
```diff
@@ -164,66 +227,95 @@ Account addresses owning liquid newton and receiving staking reward revenue are:
...
+-  The validator's commission rate on delegated stake. Commission is a percentage amount deducted by a validator from staking rewards before rewards are distributed to the validator's stake delegators. The rate can be any value in the range `0 - 100%`. At registration all validators have commission set to a default rate specified by the Autonity network's genesis configuration. (See Reference [Genesis, `delegationRate`](/reference/genesis/#configautonity-object).) After registration the validator can modify its commission rate - see [Validator commission rate change](/concepts/validator/#validator-commission-rate-change) on this page.
+
+Slashing reward revenue potential is determined by the validator reporting a proven fault committed by other validators while a member of the consensus committee. This is driven by:
```
Comment (lorenzo-dev1) [inline concepts/validator/index.md]: I do not understand this sentence, is it trying to explain what are the advantages for a misbehaviour reporter? I think the advantages for reporting a faults are only that you get the rewards of the faulty validator

File: node-operators/install-aut/index.md [hunk 12]
```diff
@@ -41,7 +41,9 @@ A public-facing internet connection with static IP is required:
...
+Running an Autonity node requires that you maintain a static ip address because the ip address forms part of the node's network address, the [enode URL](/glossary/#enode). I.e. the IP address should not dynamically change due to resetting by the IP provider.
+
+The enode provides the network location of the node client for p2p networking. Changing the ip address will change the node's identity on the network, therefore. This is significant if you are operating a validator node as the enode is provided as part of the validator meta data when [registering a validator](/validators/register-vali/#step-3.-submit-the-registration-transaction).
```
Comment (lorenzo-dev1) [inline node-operators/install-aut/index.md]: Maybe here we could add a link to the validator migration?

File: overview/index.md [hunk 13]
```diff
@@ -76,3 +76,4 @@ In addition to Ethereum concepts such as `address`, `gas`, and `node`, the disti
 | [Auton Stabilization Mechanism (ASM)](/concepts/asm/) | The protocol mechanism used to maintain [Auton](/concepts/protocol-assets/auton/) price stability. The Stabilization Mechanism is CDP, [Auton](/concepts/protocol-assets/auton/) borrowed against [Newton](/concepts/protocol-assets/newton/) or [Liquid Newton](/concepts/protocol-assets/liquid-newton/) collateral. |
 
 For all Autonity terminology and concepts see the [Glossary](/glossary/).
+Autonomous Currency Unit
```
Comment (lorenzo-dev1) [inline overview/index.md]: I saw this string added at the end of multiple files, is this intended?

File: validators/migrate-vali/index.md [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+description: >
+  How to migrate an Autonity validator node to a new instance or to a new IP/Port address
+---
+
+::: {.callout-note title="Prerequisites" collapse="false"}
+- Ensure that the host machine meets the [minimum requirements](/node-operators/install-aut/#requirements)
+:::
+
+
+## Migrating validator node to a new environment
```
Comment (lorenzo-dev1) [inline validators/migrate-vali/index.md]: I think it is a bit confusing that this part which is on top of the page links to the /node-operators/migrate-aut page which says you cannot change your IP.

I think people will get confused if they see this part first. I would move it down unless I am missing something here

**Review by lorenzo-dev1 — COMMENTED** [review]
Overall great work and LGTM. Left some comments

File: overview/index.md (hunk 13 above)
Comment (jay-cousins) [reply overview/index.md]: No!  Found and removed!

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.

File: concepts/consensus/pos/index.md (hunk 6 above)
Comment (jay-cousins) [reply concepts/consensus/pos/index.md]: Done.

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.

File: concepts/system-model/index.md (hunk 7 above)
Comment (jay-cousins) [reply concepts/system-model/index.md]: Yep, less is more. 

I've removed the repetition (Deleting: The corresponding public keys are accessible to other participants and a participant can identify other participants by public key, therefore. The _public node key_ is used to derive participant identifier and network addresses. Namely, a unique network address (the [enode URL](/concepts/validator/#validator-enode-url)), and a _node address_ for the validator (the public key for the private node key).  The _public consensus key_ is used to identify the validator during consensus participation.).

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.

File: concepts/system-model/index.md (hunk 8 above)
Comment (jay-cousins) [reply concepts/system-model/index.md]: Corrected to 'increase or decrease'.

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.

File: concepts/validator/index.md (hunk 9 above)
Comment (jay-cousins) [reply concepts/validator/index.md]: Corrected to signing. (I'll look for instances elsewhere, too.)

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.

File: concepts/validator/index.md (hunk 10 above)
Comment (jay-cousins) [reply concepts/validator/index.md]: Corrected.

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.

File: concepts/validator/index.md (hunk 11 above)
Comment (jay-cousins) [reply concepts/validator/index.md]: Hmm. I've simplified this to:

>Slashing rewards can be earned by a validator for reporting a slashable fault committed by another committee member per the [accountability and fault detection](/concepts/accountability/) protocol. As reward the _reporting validator_ is awarded the staking rewards the _offending validator_ would have earned for the epoch.

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.

File: node-operators/install-aut/index.md (hunk 12 above)
Comment (jay-cousins) [reply node-operators/install-aut/index.md]: Yes, done.

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.

File: validators/migrate-vali/index.md (hunk 14 above)
Comment (jay-cousins) [reply validators/migrate-vali/index.md]: I've switched the order, and it looks better. I think you're right  on 'confusion'. 

(I'm wondering if it could be removed altogether from this page. That can be done in a follow on PR, if needs be.)

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.

File: concepts/validator/index.md (hunk 11 above)
Comment (lorenzo-dev1) [reply concepts/validator/index.md]: much clearer 👍🏻

**Review by lorenzo-dev1 — APPROVED** [review]
LGTM now!
